pom.xml

```diff
@@ -459,14 +459,14 @@
 				</exclusions>
 			</dependency>
 			<dependency>
-				<groupId>junit</groupId>
-				<artifactId>junit</artifactId>
-				<version>4.13</version>
+				<groupId>org.junit.vintage</groupId>
```

Comment:
Good , did u see the coveralls decrease?, whay did u include junit.vintage,

Reply:
@sherrif10 , the decrease seems too tiny , so we an just ignore it

Reply:
@mozzy11  Is there any possibility that org.junit.vintage groupid might be reducing the coveralls.

Reply:
@sherrif10  is for the backward compatibility . do you have any suggestion ??